my-sms/app/models/institute.rb

```diff
@@ -8,4 +8,8 @@ class Institute < ActiveRecord::Base
   validates :name, presence: true
 
   DEFAULT_PER_PAGE = 10
+
+  def to_s
+    name
+  end
```

Comment:
can move to decorator

Reply:
Fixed in 4dac98b60e980e47b79aec7dbc42c7592b468c33

Also uncovered the issue that I hadn't set up decoration on course/institute association.